suite-native/assets/src/components/Assets.tsx

```diff
@@ -51,11 +57,17 @@ export const Assets = () => {
         setSelectedAssetSymbol(null);
     }, []);
 
+    const navigateToAssets = () => {
+        navigation.navigate(AppTabsRoutes.AccountsStack, { screen: AccountsStackRoutes.Accounts });
+    };
+
+    const isViewMoreButtonVisible = assetsDataWithPercentage.length > 3;
+
     return (
         <>
             <Card>
                 <VStack spacing={19}>
-                    {assetsDataWithPercentage.map(asset => (
+                    {assetsDataWithPercentage.slice(0, 3).map(asset => (
```

Comment:
nit: since the assetsDataWithPercentage is only used here, you could already return just the first 3 assets from the selector (so the percentage doesn't get calculated for all of them since its pointless) and just check length of assets from `assetsData`

Reply:
constants file created in 019dcf358